### PR TITLE
fix: change colors of today and free keywords in event details

### PR DESCRIPTION
**Pull request by PekkaKela**
### Closes

**[DEV-XXX](https://helsinkisolutionoffice.atlassian.net/browse/DEV-XXX):**

### CODE REVIEW

**Comment by terovirtanen** [comment]
**Events-Helsinki branch is deployed to platta: https://tapahtumat-pr301.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Hobbies-Helsinki branch is deployed to platta: https://harrastukset-pr301.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://harrastukset-pr301.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**VENUE GRAPHQL PROXY is deployed to platta: https://venue-graphql-proxy-pr301.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __failed__ for https://tapahtumat-pr301.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**EVENTS GRAPHQL PROXY is deployed to platta: https://events-graphql-proxy-pr301.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Sports-Helsinki branch is deployed to platta: https://liikunta-pr301.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://liikunta-pr301.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __failed__ for https://tapahtumat-pr301.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
## TestCafe result is __failed__ for https://tapahtumat-pr301.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tapahtumat-pr301.dev.hel.ninja :laughing::tada::tada::tada:

**Review by nikomakela — APPROVED** [review]

File: packages/components/src/components/keyword/keyword.module.scss [hunk 1]
```diff
@@ -20,4 +20,12 @@
       color: var(--color-white);
     }
   }
+
+  &.isToday {
+    background-color: var(--color-summer-medium-light);
+  }
+
+  &.isFreeEvent {
+    background-color: #afe4de;
```
[the remark this answers is not in the document]
Comment (PekkaKela) [reply packages/components/src/components/keyword/keyword.module.scss:29]: I can check if it is easy to set as variable and if it is will do it

[the remark this answers is not in the document]
Comment (PekkaKela) [reply packages/components/src/components/keyword/keyword.module.scss]: Added color as variable --color-keyword-free-event

**Review by nikomakela — APPROVED** [review]

**Comment by terovirtanen** [comment]
**Hobbies-Helsinki branch is deployed to platta: https://harrastukset-pr301.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://harrastukset-pr301.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**Events-Helsinki branch is deployed to platta: https://tapahtumat-pr301.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Sports-Helsinki branch is deployed to platta: https://liikunta-pr301.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**VENUE GRAPHQL PROXY is deployed to platta: https://venue-graphql-proxy-pr301.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tapahtumat-pr301.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://liikunta-pr301.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**EVENTS GRAPHQL PROXY is deployed to platta: https://events-graphql-proxy-pr301.dev.hel.ninja** :rocket::rocket::rocket:

**Pull request merged**